sstables/mx/writer.cc

```diff
@@ -608,6 +608,7 @@ class writer : public sstable_writer::writer_impl {
     } _pi_write_m;
     utils::UUID _run_identifier;
     bool _write_regular_as_static; // See #4139
+    bool _cannot_have_partition_tombstones = true;
```

Comment:
cannot_have_partition_tombstones is hard to parse to me. what about renaming it to contains_partition_tombstones and setting it to false by default?